readalongs/text/convert_xml.py

```diff
@@ -139,6 +136,13 @@ def convert_word(word: str, lang: str):
                 ) from e
             tg = converter(word)
             text = tg.output_string
+            if not text:
+                nonlocal g2p_empty_warning_count
```

Comment:
ohhh this is handy - I didn't actually know about this. I thought the only options were local and global. cool!